script.game.netwalk/addon.xml

```diff
@@ -1,9 +1,11 @@
 <?xml version="1.0" encoding="utf-8" standalone="yes"?>
-<addon id="script.game.netwalk" name="Netwalk Game" version="0.0.5" provider-name="Tristan Fischer (sphere@dersphere.de)">
+<addon id="script.game.netwalk" name="Netwalk Game" version="0.0.6" provider-name="Tristan Fischer (sphere@dersphere.de)">
     <requires>
-        <import addon="xbmc.python" version="2.1.0"/>
+        <import addon="xbmc.python" version="2.14.0"/>
     </requires>
-    <extension point="xbmc.python.script" library="game.py"/>
+    <extension point="xbmc.python.script" library="game.py">
+        <provides>game</provides>
```

Comment:
@da-anda have you tested if this works on older Kodi/XBMC versions?

Reply:
will double check when I work on those two extensions again. Been a bit busy getting the terrace/deck done before temperature drops below zero at night